influxql/ast_test.go

```diff
@@ -258,7 +258,7 @@ func TestSelectStatement_HasWildcard(t *testing.T) {
 
 		// No GROUP BY wildcards, time only
 		{
-			stmt:     `SELECT mean(value) FROM cpu GROUP BY time(5ms)`,
+			stmt:     `SELECT mean(value) FROM cpu where time < now() GROUP BY time(5ms)`,
```

Comment:
@pauldix this is the parser only.  It would certainly fail once it hit the query engine, which is a different test.